pkg/ipc/ipc.go

```diff
@@ -154,6 +156,45 @@ func FlagsToSandbox(flags EnvFlags) string {
 	return "none"
 }
 
+// nolint: gocyclo
+func FeaturesToFlags(feat *host.Features, manual csource.Features) EnvFlags {
```

Comment:
(not an action request, just reflecting)

It's good that we have these strings consts in fewer places now, but still both the usage of these strings directly in several packages and the `csource.Features` and `host.Features` being defined differently and independently from each other calls for further refactoring here.

Reply:
It's worse: they are also duplicated in csource.Options, in executor, in flags of multiple binaries and probably somewhere else.
I've tried to fix it all here:
https://github.com/dvyukov/syzkaller/commit/fc4b8fb553764bcbaabfbca4fbff806d4c558b12
and here:
https://github.com/dvyukov/syzkaller/commits/dvyukov-features-prepare/
but never finished (too many changes at once, maybe easier in smaller steps).
But the idea was to have all of these in a single base package sys/features that everybody can depend on, and features will have attached meta info (flag name, macro name, supported OSes, etc):
https://github.com/dvyukov/syzkaller/commit/fc4b8fb553764bcbaabfbca4fbff806d4c558b12#diff-889dde93b16b57266c6ad1513c8b8f6c605aa755d9087c8c9b9b2c1c5bcc9478R19